src/main/java/com/ldtteam/structurize/client/BlueprintRenderer.java

```diff
@@ -618,6 +626,45 @@ public BufferBuilder builder(RenderType renderType)
             }
             return buffer;
         }
+    }
 
+    public static class TransparencyHack
```

Comment:
Please doc this here, case we someday might have to port this without you =D

Reply:
meh k :D